docs/tables/aws_ec2_instance.md

```diff
@@ -69,6 +69,46 @@ where
   not tags :: JSONB ? 'application';
 ```
 
+### Get maintenance option for each instance
+
+```sql
+select
+  instance_id,
+  instance_state,
+  launch_time,
+  maintenance_options ->> 'AutoRecovery' as auto_recovery
+from
+  aws_ec2_instance;
+```
+
+### Get license details for each instance
+
+```sql
+select
+  instance_id,
+  instance_type,
+  instance_state,
+  l ->> 'LicenseConfigurationArn' as license_configuration_arn
+from
+  aws_ec2_instance,
+  jsonb_array_elements(licenses) as l;
+```
+
+### List instances by placement group
```

Comment:
@ParthaI what is the importance of adding this example? Maybe we could add it under the category of placement group details of instances?